.github/workflows/browser-compatibility-test-previous-major-version.yml

```diff
@@ -311,10 +311,15 @@ jobs:
         run: npm install axios
       - name: Send Slack Message
         run: node .github/script/send-test-report.js ${{ secrets.SLACK_JS_SDK_DEV_CORE_PREV_VER_WEBHOOK }} ${{ env.WORKFLOW_URL}} ${{ env.WORKFLOW_JOBS_STATUS }}
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME_SDK_DEV }}
+          aws-region: us-east-1
       - name: Send Metric to CloudWatch
-        if: always()
```

Comment:
if removed, will this step be skipped if a previous step failed?

Reply:
Its within the context of the job - `browser-compatibility-results-report-previous-major-version` so I dont think that job should fail